<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RhytmWindow</class>
 <widget class="QWidget" name="RhytmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rhytms view</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="WidgetControl" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Rhytms settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="CheckBoxDelta">
           <property name="text">
            <string>delta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="SpinBox1_1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>48</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="SpinBox1_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="CheckBoxTheta">
           <property name="text">
            <string>theta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="SpinBox2_1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="SpinBox2_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="CheckBoxAlpha">
           <property name="text">
            <string>alpha</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="SpinBox3_1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="SpinBox3_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>13</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="CheckBoxBetha">
           <property name="text">
            <string>betha</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="SpinBox4_1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>13</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="SpinBox4_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="CheckBoxGamma">
           <property name="text">
            <string>gamma</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="SpinBox5_1">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>40</number>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="SpinBox5_2">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>48</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Chart settings</string>
        </property>
        <layout class="QVBoxLayout" name="LayoutChartSettings">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QLabel" name="LabelDuration">
           <property name="text">
            <string>Duration (sec): 20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="SliderDuration">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="sliderPosition">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LabelAmplitude">
           <property name="text">
            <string>Amplitude (uV): 20</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="SliderAmplitude">
           <property name="minimum">
            <number>20</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>20</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="WidgetCharts" native="true">
     <layout class="QVBoxLayout" name="LayoutCharts">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSlider" name="SliderChart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SliderDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_chart_redraw_request()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderAmplitude</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_chart_redraw_request()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBoxDelta</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBoxTheta</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBoxAlpha</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBoxBetha</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckBoxGamma</sender>
   <signal>stateChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox1_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox1_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox2_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox2_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox3_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox3_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox4_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox4_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox5_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBox5_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RhytmWindow</receiver>
   <slot>_rhytms_param_cnd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_chart_redraw_request()</slot>
  <slot>_rhytms_param_cnd()</slot>
 </slots>
</ui>
